package-lock.json

```diff
@@ -18,7 +18,7 @@
         "node-fetch": "3.1.1"
```

Comment:
More unrelated changes

Reply:
Sorry new to git. Might be due to accidentely updated the @bael/eslint-parser...